app/Providers/AppServiceProvider.php

```diff
@@ -14,7 +14,9 @@ class AppServiceProvider extends ServiceProvider
      */
     public function register()
     {
-        //
+        if ($this->app->environment() !== 'production') {
```

Comment:
Should Laravel's auto package discovery not pick this up without needing this? The composer package should be `--dev`